src/allotropy/parsers/qiacuity_dpcr/qiacuity_dpcr_structure.py

```diff
@@ -40,7 +40,7 @@ def _create_measurements(data: SeriesData) -> Measurement:
         plate_identifier=data.get(str, "Plate ID"),
         target_identifier=data[str, "Target"],
         total_partition_count=data[int, "Partitions (valid)"],
-        concentration=data[float, "Concentration (copies/µL)"],
+        concentration=data[float, "Concentration (copies/μL)"],
```

Comment:
Note to reviewer - this is because we now use the same mu character everywhere, so this file needs to use the normalized version